eos/form-factors/parametric-kkvdz2022.cc

```diff
@@ -25,7 +25,7 @@ using namespace std::literals::complex_literals;
 namespace eos
 {
     KKvDZ2022FormFactors::KKvDZ2022FormFactors(const Parameters & p, const Options & o) :
-        opt_subtracted(o, "subtracted", { "off", "on" }, "off"),
+        opt_subtracted(o, options, "subtracted"),
```

Comment:
@mreboud There is no analysis file that used this.

Reply:
Ok, but I will not clean KKvDZ in this PR. If this parametrization is still useful, it can be cleaned, otherwise we can remove it